monkey/infection_monkey/exploit/tools/helpers.py

```diff
@@ -29,11 +29,8 @@ def get_target_monkey(host):
 
     if not monkey_path:
         if host.os.get("type") == platform.system().lower():
-            # if exe not found, and we have the same arch or arch is unknown and we are 32bit,
-            # use our exe
-            if (not host.os.get("machine") and sys.maxsize < 2 ** 32) or host.os.get(
-                "machine", ""
-            ).lower() == platform.machine().lower():
+            # if exe not found, and we have the same arch, use our exe
```

Comment:
I don't understand this comment, maybe remove it?

Reply:
Looking into it.